transcendence-app/src/chat/chat.controller.ts

```diff
@@ -184,6 +186,23 @@ export class ChatController {
     return messages;
   }
 
+  @Patch('room/:chatroomId')
+  @ApiBadRequestResponse({ description: 'Bad Request' })
+  @ApiUnprocessableEntityResponse({ description: 'Unprocessable Entity' })
+  async updateChatroom(
+    @Param('chatroomId', ParseUUIDPipe) chatroomId: string,
+    @Body() updateChatroomDto: UpdateChatroomDto,
+  ): Promise<Chatroom> {
+    const updatedChatroom = await this.chatService.updateChatroom(
+      chatroomId,
+      updateChatroomDto,
+    );
+    if (!updatedChatroom) {
+      throw new UnprocessableEntityException();
+    }
```

Comment:
I think this should better a `ServiceUnavailableException`. Check the [conversation](https://github.com/42AGV/ft_transcendence/pull/302#discussion_r1012830877) I had with @gbudau

Reply:
thought: In this case, we could go with `UnprocessableEntityException` or `ConflictException` because the name is unique in the [Chatroom table](https://github.com/42AGV/ft_transcendence/blob/21d562e00022cc6436fd5bd47618c95f788be3c5/db/setup/schema.sql#L45).

And if we make the name non-unique, `ServiceUnavailable` then looks more appropriate.